keps/sig-auth/2579-psp-replacement/README.md

```diff
@@ -0,0 +1,1085 @@
+          disabling the feature is a matter of adding or removing the admission plugin from the list
+          of enabled admission plugins.
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+        - Yes. Admission plugins are statically configured, so enabling or disabling will require an
+          API server restart. This can be done in a rolling manner for HA clusters.
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+        - No. This feature does not touch any node components.
+
+* **Does enabling the feature change any default behavior?**
+  Any change of default behavior may be surprising to users or break existing
+  automations, so be extremely careful here.
+    - No.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+  Also set `disable-supported` to `true` or `false` in `kep.yaml`.
+  Describe the consequences on existing workloads (e.g., if this is a runtime
+  feature, can it break the existing applications?).
+  - Yes. Disabling it means that the policies will no longer be enforced, but there are no stateful
+    changes that will be affected.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+  - There might be pods violating the policy in namespaces when it's turned on. The [Updates
+    section](#updates) explains how this case is handled.
+
+* **Are there any tests for feature enablement/disablement?**
+  The e2e framework does not currently support enabling or disabling feature
+  gates. However, unit tests in each component dealing with managing data, created
+  with and without the feature, are necessary. At the very least, think about
+  conversion tests if API types are being modified.
+  - There will be tests for updates to "violating" pods, but I do not think explicit feature
+    enable/disablement tests are necessary.
+
+### Rollout, Upgrade and Rollback Planning
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **How can a rollout fail? Can it impact already running workloads?**
+  Try to be as paranoid as possible - e.g., what if some components will restart
+   mid-rollout?
+
+* **What specific metrics should inform a rollback?**
+
+* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+  Describe manual testing that was done and the outcomes.
+  Longer term, we may want to require automated upgrade/rollback tests, but we
+  are missing a bunch of machinery and tooling and can't do that now.
+
+* **Is the rollout accompanied by any deprecations and/or removals of features, APIs,
+fields of API types, flags, etc.?**
+  Even if applying deprecation policies, they may still surprise some users.
+
+### Monitoring Requirements
```

Comment:
While not required for alpha, I strongly encourage considering these question for alpha we can gather usage data to know if anyone even tried our alpha feature.

Reply:
Will do.